src/nvim/ex_docmd.c

```diff
@@ -3275,7 +3275,7 @@ void ex_ni(exarg_T *eap)
 {
   if (!eap->skip)
     eap->errmsg = (char_u *)N_(
-        "E319: Sorry, the command is not available in this version");
+        "E319: The command is not available in this version");
```

Comment:
I'm always in favor of removing apologies from error messages (annoying and unprofessional). This "key" will need to be changed in the `.po` files too though.